monty/exts/info/github_info.py

```diff
@@ -1111,14 +1097,15 @@ async def handle_issue_comment(self, message: disnake.Message, issues: list[Foun
                 component.label = f"View {num}{suffix} comment"
 
         components.insert(0, DeleteButton(message.author))
-        await message.reply(
+        reply_method = getattr(message, "reply", None) or message.send
+        await reply_method(
             embeds=comments,
             components=components,
             allowed_mentions=disnake.AllowedMentions(replied_user=False),
         )
 
     @commands.Cog.listener("on_message")
-    async def on_message_automatic_issue_link(self, message: disnake.Message) -> None:
+    async def on_message_automatic_issue_link(self, message: Union[disnake.Message, Any], content: str = None) -> None:
```

Comment:
`Union[..., Any]` is... yeahh. Not like typing is being checked currently anyway, but it would probably help find issues earlier regardless